apps/etterlatte-saksbehandling-ui/client/src/components/behandling/soeknadsoversikt/gyldigFramsattSoeknad/barnepensjon/Innsender.tsx

```diff
@@ -19,25 +21,29 @@ export const Innsender = ({ persongalleriGrunnlag, gjenlevendeGrunnlag, harKilde
       </InfoWrapper>
     )
   }
+  const innsender = personer?.innsender?.opplysning
   const gjenlevende = gjenlevendeGrunnlag?.opplysning
-  const oppfylt = persongalleriGrunnlag.opplysning.innsender == gjenlevende?.foedselsnummer
-  const navn = oppfylt ? [gjenlevende?.fornavn, gjenlevende?.mellomnavn, gjenlevende?.etternavn].join(' ') : 'Ukjent'
-  const tekst = settTekst(oppfylt)
+  const gjenlevendeErInnsender = persongalleriGrunnlag.opplysning.innsender == gjenlevende?.foedselsnummer
+  const soekerErInnsender = persongalleriGrunnlag.opplysning.innsender === persongalleriGrunnlag.opplysning.soeker
+
+  const navn = innsender
+    ? [innsender.fornavn, innsender.mellomnavn, innsender.etternavn].filter((navn) => !!navn).join(' ')
```

Comment:
Burde kanskje hatt en funksjon for navneting her da det blir gjort mange steder helt likt

Reply:
Ser at det fins en funksjon allerede, tar i bruk den i stedet (og fikser opp mot nyeste endringer som kom med i fiksingen av foreldreansvar)